COLLABORATOR_GUIDE.md

```diff
@@ -76,6 +76,24 @@ where a pull request:
 
 The CC Members should serve as the final arbiter where required.
 
+### Cross-coordination Guidelines to Other Governing Groups
+All Board requests should be communicated to the Individual Member Directors from the 
+CommComm Chairperson each month as resolutions or updates. Directors present
+at the Board session are responsible for championing these items.
+  
+Coordinate with the TSC Director and Chairperson to ensure all relevant CommComm items 
```

Comment:
For sentence flow, what do you think about "The CommComm Chairperson should also coordinate with..."?